src/react-app-env.d.ts

```diff
@@ -0,0 +1,2 @@
+// / <reference types="react-scripts" />
```

Comment:
Can we remove it?

https://github.com/odota/web/pull/2211#discussion_r331854185

Reply:
We require this file, but we should probably .gitignore it